x-pack/plugins/enterprise_search/public/applications/shared/kibana/kibana_logic.test.ts

```diff
@@ -33,6 +33,12 @@ describe('KibanaLogic', () => {
       expect(KibanaLogic.values.config).toEqual({});
     });
 
+    it('gracefully handles disabled security', () => {
+      mountKibanaLogic({ ...mockKibanaValues, security: undefined } as any);
+
+      expect(KibanaLogic.values.security).toEqual({});
+    });
```

Comment:
@yakhinvadim I added this test because the `props.security || {}` line was showing as uncovered in jest coverage reports.

FYI though just a flat `{}` isn't sufficient if security is disabled for the Account Settings page, that view would crash if security were actually disabled due to several missing nested obj keys.

Reply:
It's actually ok, because the entire Workplace Search becomes unavailable with security disabled. Since Workplace Search is unavailable, this page cannot be opened.

But thanks for pointing this out, I realized that this logic could really use a helpful comment. 🙌

Reply:
Added a note to our polish task: https://github.com/elastic/workplace-search-team/issues/1717